app/src/main/res/values/strings.xml

```diff
@@ -137,7 +136,7 @@
     <string name="SHOW">SHOW</string>
     <string name="notify_stream" formatted="false">" in %1$s/%2$s"</string>
     <string name="notify_private" formatted="false">from %1$s</string>
-    <string name="today_messages">Today\'s messages</string>
+    <string name="today_messages">Today\'s Messages</string>
```

Comment:
We follow this capitalisation scheme read here https://github.com/zulip/zulip-android/issues/405